org-node/src/lib.rs

```diff
@@ -372,6 +381,58 @@ async fn subscribe_events(url: String, addresses: Vec<Address>, update: mpsc::Se
     }
 }
 
+/// Read named pipe / fifo file;
+async fn sign_refs(root_dir: Option<PathBuf>, mut handle: client::Handle) {
+    // NOTE: This would be better handled in a TryFrom for the options in the main.rs file;
+    let root_dir = match root_dir {
+        Some(dir) => dir,
+        None => match std::env::var("RAD_HOME") {
+            Ok(dir) => PathBuf::from(dir),
+            Err(err) => {
+                tracing::error!(target: "org-node", "Root directory or RAD_HOME env not found: {:?}", err);
+                return;
+            }
+        },
+    };
```

Comment:
This is handled in `master` afaict?

Reply:
Perhaps worth squashing commits and rebasing on top of master now

Reply:
missed the Profile load paths, removed as extraneous.